src/js/components/SelectMultiple/__tests__/__snapshots__/SelectMultiple-test.tsx.snap

```diff
@@ -182,6 +182,7 @@ exports[`SelectMultiple children 1`] = `
   margin: 0;
   border: 1px solid rgba(0,0,0,0.33);
   border-radius: 4px;
+  border: none;
```

Comment:
Do we want this? I thought I read in the code 
`readOnly && border: none`
so why would this show up in other snapshots?

Reply:
These components are using a readOnly TextInput internally